selectrum-helm.el

```diff
@@ -6,7 +6,7 @@
 ;; Created: 15 Apr 2020
 ;; Homepage: https://github.com/raxod502/selectrum
 ;; Keywords: extensions
-;; Package-Requires: ((emacs "25.1") (helm "3.6.1") (selectrum "3.1"))
+;; Package-Requires: ((emacs "26.1") (helm "3.6.1") (selectrum "3.1"))
```

Comment:
Remove selectrum-helm now? I don't think it is up to the quality of the project. If someone really wants this, they can pick up the latest version from the repository and maintain it as a separate repository/package.

Reply:
Okay, @raxod502 has commented that he would be okay with it, too. But it can be done in a separate PR.

Reply:
Why are you always going via PRs btw?

Reply:
Maybe he has done this through the web interface. I usually go with one topic per PR than I squash the commits to have a single commit per topic and no merge commits. The respective PRs can then be used for further discussion on the respective topic.

Reply:
Ah, right you can edit via the web interface. I forgot that. It makes sense to use PRs, it definitely creates a cleaner history and you don't accumulate commits as quickly as I do for example in Consult.

Reply:
But I also see the benefit doing more succinct commits in cases like your refactorings you have done #463.

Reply:
In the past I've committed directly to master, but I decided to start making pull requests for the sake of greater transparency. I'm fine with whatever policy you like.